pull_chart_tarball.go

```diff
@@ -0,0 +1,90 @@
+package helmclient
```

Comment:
Using a separate file as helmclient.go is nearly 1k LOC.

Reply:
I personally don't mind having it even 3k loc.

Reply:
I prefer the separate file since it only relates to PullChartTarball. I think doing the same for EnsureTillerInstalled would be nice as its also a lot of LOC.